__tests__/profile/profile_header.test.js

```diff
@@ -178,21 +163,27 @@ describe('downloading facilities', () => {
     })
 
     test('download is enabled', () => {
-        let component = new Component();
-        const header = new Profile_header(component, parents, geometries, api, null, geography, config);
-        let downloadBtn = screen.getByTestId('download-btn');
-        expect(downloadBtn.classList.contains('hidden')).toBe(false);
+        const component = new Component();
+        const category = {
+            label: 'test',
+            count: 17
+        }
+        const facilityRowClone = $('body').find('.location-facility__list_item')[0].cloneNode(true);
+        let f = new FacilityItem(component, facilityRowClone, category, true, false);
+        const downloadBtn = $(f.facilityItem).find('.location-facility__item_download')[0];
+
+        expect(downloadBtn).not.toHaveClass('hidden');
     })
 
     test('download is disabled', () => {
-        config = {
-            richdata: {
-                hide: ['facility-downloads']
```

Comment:
why are we losing this config? I think it's broken because I couldn't hide the button with this config recently but the test still states that it's testing the hidden functionality.

How does this test even pass if it's the same as the one above but asserting the opposite?

Reply:
the test above passes `false` for `isDownloadsDisabled` parameter and expects `downloadBtn` to not be hidden
https://github.com/OpenUpSA/wazimap-ng-ui/blob/741ae253e7b82a98a8ddf8382c402a59ca640e68/__tests__/profile/profile_header.test.js#L172 

while the test below passes `true` and expects `downloadBtn` to be hidden
https://github.com/OpenUpSA/wazimap-ng-ui/blob/741ae253e7b82a98a8ddf8382c402a59ca640e68/__tests__/profile/profile_header.test.js#L185

Reply:
let me double check the config again. but here I test the FacilityItem object while the `config` is processed in `ProfileHeader`. that's why I removed `config` but let me add another test to confirm that the `config` element is processed correctly

Reply:
added back the config

Reply:
sounds like the config is irrelevant to the test and in the full codebase we rely on some code above new FacilityItem() to convert the config into the right parameter to FacilityItem constructor, right?